.travis.yml

```diff
@@ -33,6 +33,10 @@ cache:
 before_cache:
   - rm -f $HOME/.cache/pip/log/debug.log
 
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y unrar
+
```

Comment:
If the unrar is missing the provider will still work for any file that's not a rar file. Most of the subtitles in legendastv are rar files but there's still some zip files there. After downloading the file, the provider will check if it's a rar file (this will pass even if unrar utility is not present) and if it's a rar file, it will try to extract it. If there's no unrar utility installed, then you'll get the following exception:

`RarCannotExec: Unrar not installed? (rarfile.UNRAR_TOOL='unrar')`